apache/wsgi-py3.conf.in

```diff
@@ -158,10 +158,11 @@ Header always set Access-Control-Allow-Headers "x-requested-with, Content-Type,
 # kml file downloads
 AliasMatch ^${APACHE_ENTRY_PATH}/kml/(map\.geo\.admin\.ch_KML_\d+\.kml)$ ${KML_TEMP_DIR}/$1
 
-# KML admin
+# KML admin TODO KML
 RewriteCond %{HTTP:X-Forwarded-Proto} !https
 RewriteRule ^${APACHE_ENTRY_PATH}/admin         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
```

Comment:
@procrastinatio @boecklic can we remove those lines  ? Are they only for kml or other purpose ?

Reply:
Yes, the `admin`path may be removed